config/desktop/jammy/environments/ice-wm/config_base/packages

```diff
@@ -1,164 +1,154 @@
-anacron
 apt-config-auto-update
+anacron
 apt-xapian-index
 arandr
+at-spi2-core
 blueman
+bluetooth
 bluez
-bluez-cups
 bluez-tools
-brltty
-brltty-x11
+cairo-5c
 cifs-utils
-clipit
 colord
 compton
-conky
-conky-all
-cups
-cups-bsd
-cups-client
-cups-filters
 dbus-x11
+desktop-base
+devhelp
 dictionaries-common
 dmz-cursor-theme
 doc-base
 dunst
-evince
-evince-common
+feh
 fontconfig
 fontconfig-config
-fonts-arphic-ukai
-fonts-arphic-uming
-fonts-dejavu-core
-fonts-freefont-ttf
-fonts-guru
-fonts-guru-extra
-fonts-kacst
-fonts-kacst-one
-fonts-liberation
-fonts-nanum
-fonts-opensymbol
-fonts-stix
-fonts-symbola
-fonts-ubuntu-console
 foomatic-db-compressed-ppds
 fspanel
 gdebi
-ghostscript-x
 gist
 gnome-bluetooth
 gnome-font-viewer
-gnome-orca
 gnome-power-manager
+gnome-software
+gnome-software-common
+gnome-software-doc
 gnome-system-monitor
 gnome-system-tools
+grun
+icewm
+icewm-common
+icewm-experimental
 gromit
 grun
 gsetroot
 gstreamer1.0-packagekit
 gstreamer1.0-plugins-base-apps
 gstreamer1.0-pulseaudio
+gstreamer1.0-tools
+gtk2-engines
+gtk2-engines-murrine
+gtk2-engines-pixbuf
+gvfs
 gvfs-backends
-hplip
-hunspell-en-us
-icewm
-icewm-common
-icewm-experimental
 inputattach
 inxi
 keyutils
-laptop-detect
-libanyevent-i3-perl
 libatk-adaptor
+libaudio2
 libev4
+libflatpak0
 libfont-afm-perl
 libfontconfig1
 libfontembed1
 libfontenc1
 libgail-common
-libgl1-mesa-dri
+libgdk-pixbuf2.0-bin
 libgnome-bluetooth13
 libgsettings-qt1
 libjson-xs-perl
-libnotify-bin
-libproxy1-plugin-gsettings
-libproxy1-plugin-networkmanager
+liblcms2-utils
+libgail-common
+libgtk-3-bin
+libpam-gnome-keyring
+libpulse0
+librsvg2-bin
+librsvg2-common
+libsndio7.0
 libu2f-udev
 libwmf0.2-7-gtk
 libxapp1
 libxcb-cursor0
 libxcursor1
-lightdm
-lightdm-gtk-greeter
-lightdm-settings
 ltpanel
 lxappearance
+lightdm
+lightdm-settings
+lxtask
```

Comment:
these were just moved, they _were_ in alphabetical order, could rather move `lxtask` to maintain that order